doozerlib/assembly_inspector.py

```diff
@@ -45,6 +45,19 @@ def __init__(self, runtime: Runtime, brew_session: ClientSession = None, lookup_
             else:
                 self._release_image_inspectors[image_meta.distgit_key] = None
 
+        # Preprocess rpm_deliveries group config
+        # This is mainly to support weekly kernel delivery
+        self._rpm_deliveries: Dict[str, Dict[str, Any]] = {}  # Dict[package_name] => per package rpm_delivery config
+        if self.runtime.group_config.rpm_deliveries:
+            for entry in self.runtime.group_config.rpm_deliveries:
+                packages = entry.packages
+                if not packages:
+                    raise ValueError("`packages` in `group_config.rpm_deliveries` can't be empty")
+                if not (entry.stop_ship_tag or entry.ship_ok_tag):
+                    continue  # no validation needed
```

Comment:
what would be the point in having an entry like this? seems likely to be a mistake (e.g. typo)

Reply:
Good point! This is a leftover from the past when kernel team hasn't started to apply stop_ship_tag and ship_ok_tag. This need to be change to raise an Error.